relay.go

```diff
@@ -323,6 +325,7 @@ func (r *Relayer) Receive(f *Frame, fType frameType) (sent bool, failureReason s
 	}
 	select {
 	case r.conn.sendCh <- f:
+		sent = true
```

Comment:
do you need this variable? the `default` case returns `false`, so the only way to get to the `return sent` is if it goes through this path, so seems like line 369 could just be `return true`?